README.md

```diff
@@ -17,82 +17,125 @@
 * [Subscriptions](#creating-subscriptions)
     * [JSON Example](#subscription-json-example)
     * [GO Example](#creating-subscription-golang-example)
-* [Events](#events)
+* [Rest API](#rest-api)
+* [Cloud Native Events](#cloud-native-events)
   * [Event via sdk](#publisher-event-create-via-go-sdk)
   * [Event via rest api](#publisher-event-create-via-rest-api)
 * [Metrics](#metrics)
 * [Plugin](#plugin)
   
 ## Event Transporter
-Cloud event proxy support two types of  transporter protocol
+Cloud event proxy support two types of transport protocol
 1. AMQ Protocol 
 2. HTTP Protocol
-#### AMQ Protocol 
+
+### AMQ Protocol 
  AMQ protocol required qdr router deployed and running manually or via AMQ interconnect operator.
  Have `transport-host` variable set to AMQ instance example `amq:localhost:5672`
 The Producer will be sending  events to bus address specified by the  publisher and consumers connect to those bus address.
 AMQ producer example
-```json
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
-            - "--transport-host=amqp://router.router.svc.cluster.local"
-            - "--api-port=8089"
+            - "--transport-host=amqp://amq-router.amq-router.svc.cluster.local"
+            - "--api-port=9085"
 ```
 
 AMQ consumer example              
-```json
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
-            - "--transport-host=amqp://router.router.svc.cluster.local"
-            - "--api-port=8089"
+            - "--transport-host=amqp://amq-router.amq-router.svc.cluster.local"
+            - "--api-port=9085"
 ```
-#### HTTP Protocol
-##### Producer
-CloudEvents HTTP Protocol will be enabled based on  url in `transport-host`.
+### HTTP Protocol
+#### Producer
+CloudEvents HTTP Protocol will be enabled based on url in `transport-host`.
 If HTTP is identified then the publisher will start a publisher rest service, which is accessible outside the container via k8s service name.
-The Publisher service will have the ability to register  consumer endpoints to publish events.
+The Publisher service will have the ability to register consumer endpoints to publish events.
 
-HTTP producer example  
-```json
+The transport URL is defined in the format of 
+```yaml
+- "--transport-host=$(TRANSPORT_PROTOCAL)://$(TRANSPORT_SERVICE).$(TRANSPORT_NAMESPACE).svc.cluster.local:$(TRANSPORT_PORT)"
+```
+
+HTTP producer example
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
             - "--transport-host=ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
```

Comment:
ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043"
or 
with first sub string of node name speperated by "." delimiter example 
ptp-event-publisher-service-cnfde22.openshift-ptp.svc.cluster.local:9043"